fe/fe-core/src/main/java/org/apache/doris/nereids/cost/CostModelV1.java

```diff
@@ -191,7 +191,7 @@ public Cost visitPhysicalDistribute(
             return CostV1.of(
                     0,
                     0,
-                    childStatistics.getRowCount() * Math.pow(beNumber, 0.5));
```

Comment:
In cost model v1, we use row count as the unified metrics to measure the cost. So here we would better to also use row-count based cost system, if we need to distinguish the distribute cost for different cases, we can use a factor parameter using dataSize as the input with a minimal value 1.